src/test/kotlin/uk/gov/dluhc/printapi/config/IntegrationTest.kt

```diff
@@ -56,7 +60,11 @@ private val logger = KotlinLogging.logger {}
  */
 @SpringBootTest(
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
-    classes = [IntegrationTest.IntegrationTestConfiguration::class],
+    classes = [
+        PrintApiApplication::class,
+        TestRestController::class, TestScheduledJob::class, TestSqsListener::class,
```

Comment:
Presumably we no longer need TestRestController::class, TestScheduledJob::class, TestSqsListener::class,
in which case the classes can be deleted and this IntegrationTest class reverted and make no changes to it?